WebService/v1/Models/SimulationApiModel/DeviceModelSimulationScriptOverride.cs

```diff
@@ -0,0 +1,72 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.Azure.IoTSolutions.DeviceSimulation.Services.Models;
+using Newtonsoft.Json;
+
+namespace Microsoft.Azure.IoTSolutions.DeviceSimulation.WebService.v1.Models.SimulationApiModel
+{
+    // SEE: <DeviceModelApiModel.DeviceModelSimulationScript> for the original fields being overridden
+    public class DeviceModelSimulationScriptOverride
+    {
+        // Optional, used to change the script used
+        [JsonProperty(PropertyName = "Type", NullValueHandling = NullValueHandling.Ignore)]
+        public string Type { get; set; }
+
+        // Optional, used to change the script used
+        [JsonProperty(PropertyName = "Path", NullValueHandling = NullValueHandling.Ignore)]
+        public string Path { get; set; }
+
+        // Optional, used to provide input parameters to the script
+        [JsonProperty(PropertyName = "Params", NullValueHandling = NullValueHandling.Ignore)]
+        public object Params { get; set; }
```

Comment:
Consider adding validation using a method provided by the "path" function.

Reply:
:+1: added TODO and github issue